### ISSUE TITLE
Experiment Repository Creation Request for Ground Water Engineering Lab NITK

### ISSUE THREAD

**Issue by SOLVE-NITK** [virtual-labs/engineers-forum#798]
## *Repository Creation Request*

Use this to get your experiment repository created or updated on GitHub under Virtual Labs organization.

1. #### Coordinating Institute: NITK Surathkal

2. #### Approver’s Name: Manabendra Saharia

3. #### Approved Proposal:  [GroundWater.pdf](https://github.com/virtual-labs/engineers-forum/files/9250866/GroundWater.pdf)

4. #### Hosted url: -----

5. #### Primary GitHub Handle Details: 
   ##### Name: SOLVE_NITK
   ##### GitHub Handle: @SOLVE-NITK
   ##### Email id: [csd@nitk.edu.in](mailto:csd@nitk.edu.in) , [dev.solve@nitk.edu.in](mailto:dev.solve@nitk.edu.in)

6. #### Secondary GitHub Handle Details: 
   ##### Name: Akshaya Acharya
   ##### GitHub Handle: @AkshayaAcharya
   ##### Email id: [acharya.akshaya04@gmail.com](mailto:acharya.akshaya04@gmail.com)

7. #### Secondary GitHub Handle Details: 
   ##### Name: Anusha B Salian
   ##### GitHub Handle: @AnushaSalian
   ##### Email id: [saliananusha1994@gmail.com](mailto:saliananusha1994@gmail.com)

**Comment by pavanchow** [comment]
@SOLVE-NITK   Please find the links for **Ground Water Engineering Lab** experiments and populate the repositories with sources of the experiments as detailed in Step 4 of the  [migration document](https://github.com/virtual-labs/engineers-forum/blob/master/migration-process.md).

1. Percolation test of soil - https://github.com/virtual-labs/exp-percolation-test-nitk
2. Infiltration test using simple (tube type) infiltrometer - https://github.com/virtual-labs/exp-simple-infiltrometer-nitk
3. Infiltration test using double-ring infiltrometer - https://github.com/virtual-labs/exp-double-ring-infiltrometer-nitk
4. Determination of transmissivity of the aquifer - https://github.com/virtual-labs/exp-transmissivity-aquifer-nitk
5. Determination of hydraulic conductivity of the aquifer - https://github.com/virtual-labs/exp-hydraulic-conductivity-aquifer-nitk
6. Determination of specific yield of open well - https://github.com/virtual-labs/exp-specific-yield-open-well-nitk
7. Geo-resistivity test for Groundwater level identification - https://github.com/virtual-labs/exp-groundwater-level-identification-nitk
8. Geo-resistivity test for bedrock identification - https://github.com/virtual-labs/exp-bedrock-identification-nitk
9. Subsurface investigation using ground penetrating radar - https://github.com/virtual-labs/exp-subsurface-investigation-nitk
10. Groundwater level identification using ground penetrating radar - https://github.com/virtual-labs/exp-ground-penetrating-radar-nitk

**Comment by SOLVE-NITK** [comment]
@pavanchow 
Requesting you to create a new repository for the following experiment : Determination of hydraulic conductivity of the aquifer - Falling head test. This experiment comes under Ground water engineering lab.

**Comment by ravikiran2020** [comment]
@SOLVE-NITK The experiment 5.Determination of hydraulic conductivity of the aquifer - https://github.com/virtual-labs/exp-hydraulic-conductivity-aquifer-nitk is the same.

**Comment by SOLVE-NITK** [comment]
@ravikiran2020 
Yes sir. But we would like to add one more experiment under ground water engineering lab that is _Determination of hydraulic conductivity of the aquifer using **Falling head test**_
Below experiment is _Determination of hydraulic conductivity of the aquifer using **constant head test**_
5.Determination of hydraulic conductivity of the aquifer - https://github.com/virtual-labs/exp-hydraulic-conductivity-aquifer-nitk

**Comment by SOLVE-NITK** [comment]
@ravikiran2020

Requesting you to create a new repository for the following experiment : Determination of hydraulic conductivity of the aquifer using Falling head test. This experiment comes under Ground water engineering lab. This is the an additional experiment.

**Comment by ravikiran2020** [comment]
@SOLVE-NITK Please find the link for Ground Water Engineering Lab experiment and populate the repositories with sources of the experiment as detailed in Step 4 of the [migration document](https://github.com/virtual-labs/engineers-forum/blob/master/migration-process.md).

11. Determination of hydraulic conductivity of the aquifer using Falling head test-https://github.com/virtual-labs/exp-hydraulic-conductivity-aquifer-falling-head-test-nitk

**Comment by SOLVE-NITK** [comment]
Thank you @ravikiran2020
